modules/plugin/jdbc/jdbc-db2/src/test/java/org/geotools/data/db2/DB23DOnlineTest.java

```diff
@@ -16,11 +16,21 @@
  */
 package org.geotools.data.db2;
 
+import org.geotools.data.simple.SimpleFeatureCollection;
+import org.geotools.geometry.jts.ReferencedEnvelope3D;
 import org.geotools.jdbc.JDBC3DOnlineTest;
 import org.geotools.jdbc.JDBC3DTestSetup;
+import org.geotools.util.factory.Hints;
+import org.opengis.filter.spatial.BBOX3D;
 
 public class DB23DOnlineTest extends JDBC3DOnlineTest {
 
+    @Override
+    protected void connect() throws Exception {
+        Hints.putSystemDefault(Hints.FORCE_LONGITUDE_FIRST_AXIS_ORDER, Boolean.TRUE);
```

Comment:
These kind of tests can make the build very brittle (as other tests running concurrently with this one may get confused as their environment changes). When running the test as part of a maven build the referencing system may already be setup using the default axis order policy.

If you absolutely need this kind of thing, [WMSCoverageReaderTest](https://github.com/geotools/geotools/blob/master/modules/extension/wms/src/test/java/org/geotools/ows/wms/map/WMSCoverageReaderTest.java) uses:

```
    @Before
    public void setup() {
        System.setProperty("org.geotools.referencing.forceXY", "true");
        Hints.putSystemDefault(Hints.FORCE_AXIS_ORDER_HONORING, "http");
        CRS.reset("all");
    }

    @AfterClass
    public static void teardown() {
        System.clearProperty("org.geotools.referencing.forceXY");
        Hints.putSystemDefault(Hints.FORCE_AXIS_ORDER_HONORING, "");
        CRS.reset("all");
    }
```

Reply:
I added a fix based on the above to your branch, hopefully it worked :)

Reply:
Thank you for applying the fix but I don't understand why this was a problem in the first place. If the DB2 online tests aren't being run, how is this becoming a problem in DB23DOnlineTest?

I thought setting System.setProperty("org.geotools.referencing.forceXY", "true") and Hints.putSystemDefault(Hints.FORCE_AXIS_ORDER_HONORING, "http") were somewhat equivalent.

Reply:
Where can I see the fix you put in?

Reply:
@jodygarnett we don't use concurrent tests in the same VM, it's an option of Maven we don't use, too many static settings around in GeoTools to go that way.